lib/fastlane_core/configuration/config_item.rb

```diff
@@ -63,16 +66,37 @@ def valid?(value)
 
     # Returns an updated value type (if necessary)
     def auto_convert_value(value)
-      # Special treatment if the user specififed true, false or YES, NO
-      if %w(YES yes true).include?(value)
-        value = true
-      elsif %w(NO no false).include?(value)
-        value = false
+      # We must cast the type to String before comparing because of the way that Ruby
+      # handles class comparisons
+      case data_type.to_s
+      when 'Array'
+        value = value.split(',')
```

Comment:
We'd need some kind of error handling here, what happens if I pass `1` as the value? I guess changing this to
```ruby
value = value.to_s.split(',')
```

should do the trick. I'd also really like to support `;` as separate for values. Is it okay to just support both?